windows/utils/win_install_online_updates.yml

```diff
@@ -54,6 +51,20 @@
   ansible.builtin.debug: var=win_updates_install_result
   when: enable_debug
 
+- name: "Restart guest OS when required"
+  when:
+    - not win_updates_install_result.failed | default(true)
```

Comment:
It's better to reboot no matter win_updates_install_result.failed is true or false.

Reply:
Ok, changed. Thanks.